cmake/sysbuild/suit.cmake

```diff
@@ -149,6 +149,57 @@ function(suit_register_post_build_commands)
   )
 endfunction()
 
+function(suit_generate_dfu_zip)
+  get_property(
+    dfu_artifacts
+    GLOBAL PROPERTY
+    SUIT_DFU_ARTIFACTS
+  )
+
+  set(script_params "")
+
+  foreach(image ${IMAGES})
+    unset(GENERATE_LOCAL_ENVELOPE)
+    sysbuild_get(GENERATE_LOCAL_ENVELOPE IMAGE ${image} VAR CONFIG_SUIT_LOCAL_ENVELOPE_GENERATE KCONFIG)
+    if(NOT DEFINED GENERATE_LOCAL_ENVELOPE)
+      continue()
+    endif()
+
+    sysbuild_get(target IMAGE ${image} VAR CONFIG_SUIT_ENVELOPE_TARGET KCONFIG)
+
+    unset(domain_app)
+    unset(domain_rad)
+    sysbuild_get(domain_app IMAGE ${image} VAR CONFIG_SOC_NRF54H20_CPUAPP KCONFIG)
+    sysbuild_get(domain_rad IMAGE ${image} VAR CONFIG_SOC_NRF54H20_CPURAD KCONFIG)
+
+    if(domain_app)
+      set(type "suit-application")
```

Comment:
Any reason for adding _suit-_ prefix here and below?

Reply:
I see that `type="application"` is already being used for other purposes in mcuboot.
https://github.com/nrfconnect/sdk-nrf/blob/main/cmake/sysbuild/zip.cmake#L100